src/server/appserver.cpp

```diff
@@ -350,6 +351,25 @@ AppServer::AppServer(int &argc, char **argv)
         }
     }
 
+    appStateValue = "";
+    if (bool found = false; !ParmsDb::instance()->selectAppState(AppStateKey::LogUploadToken, appStateValue, found) || !found) {
+        LOG_ERROR(_logger, "Error in ParmsDb::selectAppState");
+    }
+    const std::string logUploadToken = std::get<std::string>(appStateValue);
+    if (!logUploadToken.empty()) {
```

Comment:
`logUploadToken` could be initialized within the `if` statement. Same remark for `ExitCode`.

Reply:
[8438744](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/84387443f3f66535561db9fcf3ec96be88834542)